### PR TITLE
[DOCS] Add Solidity IDE to resources page

### PR DESCRIPTION
**Pull request by System-Glitch**
### Description

Add [Solidity IDE](https://github.com/System-Glitch/Solidity-IDE) to the "Resources" documentation page. (`docs/resources.rst`)

### Checklist
- [x] Code compiles correctly
- [ ] All tests are passing
- [ ] New tests have been created which fail without the change (if possible)
- [ ] README / documentation was extended, if necessary
- [ ] Changelog entry (if change is visible to the user)
- [x] Used meaningful commit messages

### CODE REVIEW

File: docs/resources.rst [hunk 1]
```diff
@@ -36,6 +36,9 @@ Solidity Integrations
     * `Superblocks Lab <https://lab.superblocks.com/>`_
         Browser-based IDE. Built-in browser-based VM and Metamask integration (one click deployment to Testnet/Mainnet).
 
+    * `Solidity IDE <https://github.com/System-Glitch/Solidity-IDE>`_
+        Browser-based IDE with integrated compiler, integrated Ganache and local file system support.
```
Comment (chriseth) [inline docs/resources.rst]: Is it really browser or electron based?

File: docs/resources.rst [hunk 2]
```diff
@@ -36,6 +36,9 @@ Solidity Integrations
     * `Superblocks Lab <https://lab.superblocks.com/>`_
         Browser-based IDE. Built-in browser-based VM and Metamask integration (one click deployment to Testnet/Mainnet).
 
+    * `Solidity IDE <https://github.com/System-Glitch/Solidity-IDE>`_
```
Comment (chriseth) [inline docs/resources.rst]: This list should be sorted alphabetically, so please move it above Superblocks - actually could you also sort `Solium` into its place? :)

File: docs/resources.rst (hunk 1 above)
Comment (System-Glitch) [reply docs/resources.rst]: It is a vue js interface interacting with a local nodejs server (which handles the file system and the compilation)

File: docs/resources.rst (hunk 2 above)
Comment (System-Glitch) [reply docs/resources.rst]: 120787ab80ca9d6a17c68b94cc38413e245026fa

File: docs/resources.rst (hunk 1 above)
Comment (ChrisChinchilla) [reply docs/resources.rst]: Small suggestion for tiny readability improvement

```suggestion
        Browser-based IDE with integrated compiler, Ganache and local file system support.
```

**Comment by ChrisChinchilla** [comment]
From a written word perspective, this is fine with me.

File: docs/resources.rst (hunk 1 above)
Comment (chriseth) [reply docs/resources.rst]: So do I open a local URL in a browser of my choice or how does it work?

Comment (System-Glitch) [reply docs/resources.rst]: Yes. When running `npm run ide`, the solc server and ganache are started and the IDE opens in your default browser.

**Comment by chriseth** [comment]
Nice! Can you squash your commits, please?

**Review by chriseth — APPROVED** [review]

**Pull request merged**